client/src/components/Pages/Structure/Input/StructuralElementInputAccordion.tsx

```diff
@@ -13,13 +14,49 @@ import { red, green } from "@material-ui/core/colors";
 import "./StructuralElementInputAccordion.scss";
 import { BaseStructuralElementProps } from "./StructuralElementInputProps";
 
+const useStyles = makeStyles(() => ({
+  cardHeaderAction: {
+    margin: "auto !important",
+    display: "flex",
+  },
+  cardHeaderTitleRoot: {
+    textAlign: "center",
+    fontSize: "15px !important",
+    marginRight: "10px !important",
+  },
+  cardHeaderSubTitleRoot: {
+    textAlign: "center",
+  },
+  cardAvatar: {
+    marginRight: "10px !important",
+  },
+  cardActionsRoot: {
+    padding: "0 8px 8px 16px !important",
+  },
+}));
+
 interface StructuralElementInputAccordionProps
   extends BaseStructuralElementProps {
   expanded: boolean;
   setExpanded?: React.Dispatch<React.SetStateAction<boolean>>;
   inputElements?: JSX.Element;
   validated: boolean;
+  icon: JSX.Element;
+}
+
+interface ExpandMoreProps extends IconButtonProps {
+  expand: boolean;
 }
+const ExpandMore = styled((props: ExpandMoreProps) => {
+  const { expand, ...other } = props;
+  return <IconButton {...other} />;
+})(({ theme, expand }) => ({
+  transform: !expand ? "rotate(0deg)" : "rotate(180deg)",
+  marginLeft: "auto",
+  transition: theme.transitions.create("transform", {
+    duration: theme.transitions.duration.shortest,
+  }),
+}));
```

Comment:
It looks like this pushes the height of the card box a bit, such that un-editable cards end up being shorter than editable cards. Maybe we should set a min-height on the collapsed card?

![Screen Shot 2022-10-19 at 9 25 52 AM](https://user-images.githubusercontent.com/12942397/196704375-35118981-d6de-4d33-aaa6-9cb2fa891bc7.png)

Reply:
great idea, and good catch!